Formula/portable-ruby.rb

```diff
@@ -85,6 +85,8 @@ def install
       ENV["SSL_CERT_FILE"] ||= File.expand_path("../../libexec/cert.pem", RbConfig.ruby)
       #{openssl_rb_content}
     EOS
+
+    cp_r ncurses.opt_share/"terminfo", share/"terminfo" if OS.linux?
```

Comment:
Just checking if I understand: if this ended up misinstalled into `opt_share` somehow, we can probably do `(share/"terminfo").install ncurses.opt_share/"terminfo"` so we outright move it out of `opt_share`?

Reply:
It's not "misinstalled" per se. It's because we only bottle portable-ruby and not any of the dependencies.

What I'm doing here is simply copying files installed in portable-ncurses to also be in the portable-ruby Cellar.

Moving would have the same effect in the end (we don't care after this point if the portable-ncurses install still functions, since it will be discarded), but I don't think the sandbox will allow us to remove files from other formula install directories anyway. (Or if it does allow for some reason, I don't think it should do.)

Reply:
I'd maybe do `nurses.share` instead of `opt_share` here.

Reply:
Yeah, fair point. I'll change the OpenSSL section too.

Reply:
Oh yeah, I'd totally misread this - thought you were copying from ncurses into ncurses, not into Ruby. 👍